packages/ffe-datepicker/less/calendar.less

```diff
@@ -157,3 +151,250 @@
         }
     }
 }
+
+.ffe-calendar__header {
+    @margin-top: 0;
+    @margin-bottom: 5px;
+    @margin-left: 0;
+    @margin-right: 0;
+    @margin-between: 13px;
```

Comment:
I would suggest new things we do now should be based on the new 8px grid to not increase the already huge task of converting everything in the future.